www/conf/sites-available/90-desec.api.location.var

```diff
@@ -2,9 +2,7 @@
 # Pass API requests to an API server
 ######
 location /api/ {
-    # max .5r/s to the API per IP, but 10 at once is okay. This limit may need to be increased once
-    # client applications become more powerful
-    ${PROD_ONLY} limit_req zone=perip-api burst=10 nodelay;
+    ${PROD_ONLY} limit_req zone=perip-api;
```

Comment:
I believe `PROD_ONLY` variable is only introduced in the next commit

Reply:
no, it is introduced in the same commit (in `envreplace.sh`)